nemo/collections/nlp/modules/common/megatron/token_level_encoder_decoder.py

```diff
@@ -123,6 +126,17 @@ def __init__(
         self.add_encoder = add_encoder
         self.add_decoder = add_decoder
         self.normalization = normalization
+        self.position_embedding_type = position_embedding_type
+        self.relative_attention_num_buckets = relative_attention_num_buckets
+        self.relative_attention_max_distance = relative_attention_max_distance
+
+        if self.position_embedding_type == 'absolute' or 'learned_absolute':
```

Comment:
We don't support just `absolute` yet, so maybe remove it from this check?

Reply:
And in the error raise below

Reply:
yes, removed them.